pkgs/tools/networking/qodem/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, autoconf, automake, ncurses, SDL, gpm, miniupnpc }:
+stdenv.mkDerivation rec {
+  pname = "qodem";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "klamonte";
+    repo = "qodem";
+    rev = "v${version}";
+    sha256 = "NAdcTVmNrDa3rbsbxJxFoI7sz5NK5Uw+TbP+a1CdB+Q=";
+  };
+
+  buildInputs = [ autoconf automake ncurses SDL gpm miniupnpc ];
```

Comment:
autoconf and automake should be `nativeBuildInputs`, because when cross-compiling you need the versions for the system doing the build.  You can make sure you've got everything in the right build inputs array by setting `strictDeps = true`.

Reply:
Ah, ty, I wasn't sure about that one.